projects/batfish/src/org/batfish/main/Batfish.java

```diff
@@ -3578,6 +3578,20 @@ public void pushDeltaEnvironment() {
       return configurationData;
    }
 
+   static List<Path> listAllFiles(Path configsPath) {
+      List<Path> configFilePaths;
+      try {
+         configFilePaths = Files.walk(configsPath)
+                 .filter(path -> !path.getFileName().toString().startsWith(".")
+                         && Files.isRegularFile(path))
```

Comment:
from [javadoc](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#isRegularFile(java.nio.file.Path,%20java.nio.file.LinkOption...)): "By default, symbolic links are followed and the file attribute of the final target of the link is read. If the option `NOFOLLOW_LINKS` is present then symbolic links are not followed."

So I believe this does what we want for symlinks.